package.json

```diff
@@ -30,7 +30,10 @@
     "deploy:cloudfront:prod": "SLS_DEBUG=* ROOT_DOMAIN=communitybridge.org PRODUCT_DOMAIN=easycla.communitybridge.org yarn sls deploy --stage='prod' --cloudfront=true --verbose",
     "deploy:frontend:prod": "SLS_DEBUG=* ROOT_DOMAIN=communitybridge.org PRODUCT_DOMAIN=easycla.communitybridge.org yarn sls client deploy --stage='prod' --cloudfront=true --no-confirm --no-policy-change --no-config-change --verbose",
     "deploy:invalidate:prod": "SLS_DEBUG=* ROOT_DOMAIN=communitybridge.org PRODUCT_DOMAIN=easycla.communitybridge.org yarn sls cloudfrontInvalidate --stage='prod' --region='us-east-1' --cloudfront='true' --verbose",
-    "remove:prod": "SLS_DEBUG=* yarn sls remove --stage='prod' --region='us-east-1' --verbose"
+    "remove:prod": "SLS_DEBUG=* yarn sls remove --stage='prod' --region='us-east-1' --verbose",
+    "prebuild:dev": "STAGE_ENV=dev node ./src/app/config/scripts/prefetch-ssm.js",
```

Comment:
Also, if possible, ensure `prefetch-ssm.js` is called when the developer performs a build.  So, if dev calls:

- `yarn prebuild:dev` - it should fetch SSM parameters and save to `vd-env-config.json` in the config folder.
- `yarn build` - should also call fetch SSM parameters and then build

Reply:
Done